hermes-frontend/src/main/java/pl/allegro/tech/hermes/frontend/buffer/BackupMessagesLoader.java

```diff
@@ -141,6 +160,30 @@ private boolean sendMessageIfNeeded(Message message, Optional<Topic> topic, Stri
         }
     }
 
+    private void waitOnBrokerTopicMetadata(Topic topic) {
```

Comment:
This code is run as the last startup hook after frontend is started. It is also run from the main thread, so it will be blocked until kafka is alive. I don't see any strong args right now why we should do this in another thread but I am open for them.

As for monitoring, this is not the way how we would like to monitor whether kafka is alive. When it is down then we will see logs from here in every 5s (default value) which informs that backup message loader waits for kafka to be responsive.